src/doc/trpl/no-stdlib.md

```diff
@@ -38,6 +38,7 @@ fn start(_argc: isize, _argv: *const *const u8) -> isize {
 // provided by libstd.
 #[lang = "stack_exhausted"] extern fn stack_exhausted() {}
 #[lang = "eh_personality"] extern fn eh_personality() {}
+#[no_mangle] pub extern fn rust_eh_unwind_resume() {}
```

Comment:
It looks like this only needs to be defined on 64-bit windows targets, so can this be omitted? I suspect 99% of `#![no_std]` users aren't working on Windows.